OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -508,13 +536,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </bindingDesc>
                     </physDesc>
                     
-                <!--    <history>
+              <history>
                         <origin>
-                            <origDate when="" evidence=""/>
-                        </origin>
-                        <provenance>  </provenance>
-                    </history>-->
-                    
+                            <origDate notBefore="1775" notAfter="1815" evidence="internal"/>
+                            According to <ref target="#coloph1"/>, the manuscript was written by <persName ref="PRS12986GabraKr" role="scribe"/> in the monastery of
+                            <placeName ref="LOC2297DabraL"/>, in <placeName ref="LOC5597Sawa"/>, under the patronage of <persName ref="PRS12983GabraMa" role="patron"><roleName type="title">ʾabbā</roleName> Gabra Māryām</persName>.
+                            It was written at the time of <persName ref="PRS9978Waldase"><roleName type="title">rās</roleName> Walda Śǝllāse</persName>.
+                            </origin>
+                        <provenance>
+                            The manuscript was brought to England by a British officer after the 1868 expedition by <persName ref="PRS7484Napier"/>, in all likelihood coming 
+                            from the <placeName ref="INS0101MadhaneAlam"/> collection. 
+                            It was then sold to the <placeName ref="INS0525Quaritch"/> and in 1897, after failed negotiations with the <placeName ref="INS00001BL"/>, 
+                            it entered the private collection of <persName ref="PRS9721Valerie"/> as ms. Lady Meux 4. The manuscript was studied and its textual contents edited 
+                            and translated by <persName ref="PRS2929Budge"/>, who had been asked to examine it by Lady Meux.
+                            It was finally bought by <persName ref="PRS5782JuelJen"/> at Sotheby's on 16 April 1985 (lot 392), according to the auction catalogue.
+                        </provenance>
```

Comment:
The manuscript was brought to England by a British officer after the 1868 expedition by <persName ref="PRS7484Napier"/>, and, in all likelihood, it came from .."
                          
Do we know what happened to the Lady Meux collection? @thea-m @SusanneHummel ? If not then maybe we should write.

"The subsequent owners are unknown, but the manuscript was eventually purchased by <persName ref="PRS5782JuelJen"/> at a Fine Oriental Manuscripts and Miniatures auction at Sotheby's on 16 April 1985 (lot 392)." The auction catalogue does not specify that JJ acquired it, it just says it was sold by him, the rest is implicit.

Reply:
@Gnisci Sorry, but I cannot comment on this because I don't know about the Lady Meux collection. However, should 'by' (after purchased) in your suggested sentences not be deleted?